src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -170,21 +177,25 @@ const MeasureEditor = () => {
             </ErrorText>
           )}
         </UpdateAlerts>
-        <Button
-          buttonSize="md"
-          buttonTitle="Save"
-          variant="primary"
-          onClick={() => updateMeasureCql()}
-          data-testid="save-cql-btn"
-        />
-        <Button
-          tw="ml-2"
-          buttonSize="md"
-          buttonTitle="Cancel"
-          variant="secondary"
-          onClick={() => resetCql()}
-          data-testid="reset-cql-btn"
-        />
+        {canEdit && (
+          <Button
+            buttonSize="md"
+            buttonTitle="Save"
+            variant="primary"
+            onClick={() => updateMeasureCql()}
+            data-testid="save-cql-btn"
+          />
+        )}
+        {canEdit && (
+          <Button
+            tw="ml-2"
+            buttonSize="md"
+            buttonTitle="Cancel"
+            variant="secondary"
+            onClick={() => resetCql()}
+            data-testid="reset-cql-btn"
+          />
+        )}
```

Comment:
```suggestion
        {canEdit && (
          <Button
            buttonSize="md"
            buttonTitle="Save"
            variant="primary"
            onClick={() => updateMeasureCql()}
            data-testid="save-cql-btn"
          />
          <Button
            tw="ml-2"
            buttonSize="md"
            buttonTitle="Cancel"
            variant="secondary"
            onClick={() => resetCql()}
            data-testid="reset-cql-btn"
          />
        )}
```
Can these be wrapped with the same `canEdit` check?

Reply:
@jkotanchik-SB They are in the same canEdit check now.